internal/controller/stages/verification.go

```diff
@@ -86,9 +91,10 @@ func (r *reconciler) startVerification(
 			logger.Debug("AnalysisRun already exists for Freight")
 			latestAnalysisRun := analysisRuns.Items[0]
 			return &kargoapi.VerificationInfo{
-				ID:        uuid.NewString(),
-				StartTime: ptr.To(latestAnalysisRun.CreationTimestamp),
-				Phase:     kargoapi.VerificationPhase(latestAnalysisRun.Status.Phase),
+				ID:           uuid.NewString(),
+				StartTime:    ptr.To(latestAnalysisRun.CreationTimestamp),
+				CompleteTime: latestAnalysisRun.Status.CompletedAt(),
```

Comment:
@gdsoumya AnalysisRun's CompletedAt time will be bubbled up to `VerificationInfo` and annotated to make consistent with start time